parcels/examples/example_dask_chunk_OCMs.py

```diff
@@ -300,6 +317,28 @@ def test_ofam_3D(mode, chunk_mode):
     assert(abs(pset[0].lat - 11) < 1)
 
 
+@pytest.mark.parametrize('mode', ['jit'])
+@pytest.mark.parametrize('chunk_mode', [False,
+                                        pytest.param('auto', marks=pytest.mark.xfail()),  # TODO: Needs to pass; see #904
+                                        pytest.param('specific', marks=pytest.mark.xfail())])  # TODO: Needs to pass; see #904
+def test_mitgcm_regridded(mode, chunk_mode):
+    if chunk_mode == 'auto':
+        dask.config.set({'array.chunk-size': '1024KiB'})
+    else:
+        dask.config.set({'array.chunk-size': '128MiB'})
+    field_set = fieldset_from_mitgcm_regrid(chunk_mode)
+    lons, lats = 5e5, 5e5
+
+    pset = ParticleSet.from_list(fieldset=field_set, pclass=ptype[mode], lon=lons, lat=lats)
+    pset.execute(AdvectionRK4, runtime=delta(days=1), dt=delta(minutes=5))
+    # MITgcm_regridded sample file dimensions: T=5, Xp1=401, lon=201
+    assert (len(field_set.U.grid.load_chunk) == len(field_set.V.grid.load_chunk))
```

Comment:
if that is enough for you for the test, then it's okay. I think with those chunking tests, we should really check that, if we request a specific chunking set, it is in the end also formatted the way it was requested (hence the specific dblocks, vblocks, ublocks - should rather be called zblocks (for 'depth'), yblocks (for 'lat') and xblocks (for 'lon')).